lobsterpy/plotting/__init__.py

```diff
@@ -6,14 +6,20 @@
 """
 from __future__ import annotations
```

Comment:
Surely we don't need `__future__` any more

Reply:
Ah, this is related to https://peps.python.org/pep-0563/ ? So the other option is to string-quote some of the annotations.

Ok then, the deferred evaluation does look pretty :smile:

Reply:
I did use string-quote for the trouble-causing annotations (Annotations with `|` were the issue). Thanks for this suggestion :-)